pkgs/development/lua-modules/overrides.nix

```
@@ -302,6 +302,10 @@ in
     ];
   });
 
+  luacheck = prev.luacheck.overrideAttrs (oa: {
+    meta.mainProgram = "luacheck";
```

```suggestion
    meta = oa.meta // {
      mainProgram = "luacheck";
    };
```

otherwise we lose the other meta attributes like `meta.description`, `meta.homepage`, etc.

---

Oops, good catch.